spec/models/claim_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.describe Claim, type: :model do
   subject(:example_address_attrs) { attributes_for :address }
 
-  let(:claim) { described_class.new }
+  let(:claim) { described_class.new attributes_for(:claim) }
```

Comment:
In general, this spec has changed alot because I am testing with and without the pdf file being built externally - so the code base is ready for ET1 changing over.  Once changed over, old code will be removed and this will be re simplified

Reply:
RST-1741 has been created to ensure this actually happens